README.md

```diff
@@ -49,9 +52,9 @@ import requests
 
 # This data defines a structured cloudevent
 attributes = {
-    "type": "README.sample.structured",
-    "id": "structured-event",
-    "source": "README",
+    "type": "com.readme.structured",
```

Comment:
Maybe use `com.example`, which is reserved by IANA for documentation? (and above)

Reply:
In addition, we should remove `structured` from the `type` as the CE type should be independent of however it is encoded for the HTTP request.

Something like `com.example.sampletype` would be better.